apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-examples/platform-radio-group-content-example.component.html

```diff
@@ -1,126 +1,96 @@
-<p>Reactive form Examples</p>
-<form [formGroup]="customForm" class="flex-form">
-    <fieldset fd-fieldset>
-        <p>Example 1: Compact Radio Button Group with 'default' state and with default selection</p>
-        <fdp-radio-group
-            [id]="'radio1'"
-            [name]="'radio1'"
-            [contentDensity]="'compact'"
-            [value]="'Spring'"
-            [isInline]="false"
-            formControlName="example1"
-        >
-            <fdp-radio-button *ngFor="let season of seasons" [value]="season">
-                {{ season }}
-            </fdp-radio-button>
+<h3>Use With Reactive Form</h3>
+<p>Compact Radio Button Group with 'default' State and with Default Selection</p>
+<fdp-form-group [multiLayout]="true" [formGroup]="form1">
```

Comment:
When you  want to show this reactive approach you can also set the value on the formGroup level and have object that this form represent and pass Object to the form. but this is just the idea, 

cuz it does not make much sense using FormGroup, but you are not setting #ff.formControl to the field, and therefore you are not really using FormGRoup.

Reply:
added example and unit-tests for formGroup->objectdata combination and formGroup->formControl(value) combination.